troposphere/static/js/components/projects/resources/instance/details/actions/InstanceActionsAndLinks.jsx

```diff
@@ -141,6 +145,11 @@ export default React.createClass({
                     onClick: this.onStop
                 });
                 linksArray.push({
+                    label: "Resize",
+                    icon: "fire",
```

Comment:
And there are two places to set the icon. I am confused and not sure why I need to have two. I put the `repeat` for the other one since `fire` doesn't work for it.